templates/manager/report-view.html

```diff
@@ -0,0 +1,78 @@
+{% extends 'base.html' %}
+{% load widget_tweaks %}
+{% load filters %}
+{% block page-body %}
+<h1 class="h2 heading-underline">Email Send Statistics</h1>
+<div class="row my-5">
+  <div class="col-lg-5 col-xl-4">
+    <div class="card">
+      <div class="card-block">
+        <div class="card-title">
+          <h2 class="h4">Report Filters</h2>
+        </div>
+        <div class="card-body">
+          <form class="form-horizontal" action="" method="get">
+            <fieldset>
+              <input type="hidden" name="action" value="url_report">
+              {% for error in form.non_field_errors %}
+                <div class="alert alert-error">
+                  {{error}}
+                </div>
+              {% endfor %}
+              <div class="form-group">
+              {% for field in form %}
```

Comment:
Consider adding field descriptions above or underneath these inputs, at least maybe for the Day of Week, URL Filter, and Email Domain fields--those fields might not be immediately understandable at first glance without additional information